### ISSUE TITLE
Community findings from the WCAG 2.2 Design Sprint 2023

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#3250]
## What
We developed two community engagement proposals as part of the [2023 WCAG 2.2 Design Sprint](https://docs.google.com/document/d/1nSp_EZW3BBZyASXgg05sla8N-WQa-GD3Hmdyf0tE0qM/edit)

### 1. WCAG 2.2 community support events
These events are proposed to focus on offering a series of [profession-based support events to help teams and services transform to WCAG 2.2](https://deploy-preview-2955--govuk-design-system-preview.netlify.app/accessibility/wcag-2-2-support-events). 

_These are different from the WCAG 2.2 inspection workshops that are happening throughout the same time period as this work._

### 2. WCAG 2.2 peer community support
This engagement proposes to create [profession-based support groups to help our community to update components in their services to WCAG 2.2](https://deploy-preview-2955--govuk-design-system-preview.netlify.app/accessibility/wcag-2-2-peer-community-support). The aim is to “Help one another out”.

## Epic Lead
TBC 

## Why

We ran some small scale user research sessions to get feedback on all the prototypes developed in the Design Sprint. [Findings and further work to the website prototype are being tracked and maintained in another EPIC](link).

We are currently choosing to prioritise the website updates Oct 2023.

The user research findings specific to the community engagement proposals have more general feedback that we could apply to other community engagement work within the Design System. This EPIC hopes to capture that as well.

## Who needs to work on this
…
## Who needs to review this
…

**Comment by CharlotteDowns** [comment]
## User research findings 

[A full list of findings recorded in Padlet](https://govukdesignsystem.padlet.org/charlottedowns1/design-sprint-user-research-findings-n7d15v9de1ir9w7x).

### WCAG 2.2 community support events
> P5 – [Community events] Would like to know more about what specifically elements they will be looking at - more details about what the thing is so they don’t end up doing the same course multiple times
> P1 – This is going to be relevant to me very infrequently (Events)
> P1 – Oooh that’d be useful if was just for content (profession based)
> P4 – events are the kind of thing we get to when we can but it's not always possible
> P3 – I ask my team if I can join these things, they always say yes

### WCAG 2.2 peer community support
> P1 – [Peer community support events] “It sounds like an investment of time… I know it sounds cynical. Am I going to put up with a whole new workstream for myself?
> P3 – Maybe it's a fortnightly meeting where people can ask specific WCAG questions
> P3 – it has potential, almost like a hackathon, does sound quite fun, can often get bogged down, lack of time or aren't that productive
> P3 – I would join a WCAG 2.2 support event

### About events/community collaborations we have ran to date
> P3 – Also seems to have very long gaps between sessions
> P3 – Task list was a bit more productive, but found we had 2 or 3 people drive it, it wasn't quite as collaborative as I’d hoped – I missed a session and found a lot of the decisions had been made
> P3 – Maps and task list, I don't find them very productive at all, Maps gets bogged down in people talking about their specific situation and getting theo point across
> P3 – At MOJ we found having a hackathon wasn't a good idea without a problem to solve
